vignettes/previewerReporter.Rmd

```diff
@@ -7,21 +7,21 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-Reporter Previewer is a shiny module to visualize and manipulate the already added report Cards, and eventually downloading a report document.
-Reporter Previewer is extended by the base modules introduced in the `simpleReporter` vignette.
+Reporter previewer is a `shiny` module to visualize and manipulate the already added report cards, and eventually downloading a report document.
```

Comment:
The module previews reports. The module is a previewer of reports. It is a report previewer.

Reply:
this was taken directly from the function name, e.g. 'reporter_previewe_ui/srv', so I thought it was referring to the class 'Reporter'. This was  confusing to me.

Reply:
[44e02fb](https://github.com/insightsengineering/teal.reporter/pull/247/commits/44e02fb0cf35f843efb574e5256cace5825cc4ac)